.github/environment/Production

```diff
@@ -16,6 +16,10 @@ react-app-gtm-container-id=GTM-M5HZBRZ
 react-app-sentry-dsn=https://8cd3c59be8ec4118aa6f6c59487b9e56@o850903.ingest.sentry.io/5893160
 ses-region=us-east-1
 sls-stage=production
-squidex-app-name=asap-hub
 squidex-backup-bucket-name=asap-hub-production-data-backup
-squidex-client-id=asap-hub:default
+crn-squidex-app-name=asap-hub
+crn-squidex-ci-client-id=asap-hub:default
+crn-squidex-api-client-id=asap-hub:default
+gp2-squidex-app-name=gp2-hub
+gp2-squidex-ci-client-id=gp2-hub:default
```

Comment:
didn't we say we would use different clients for API and CI for GP2?

Reply:
yes, but we don't have production yet. They are more placeholders.